k8s/aat/common/ia/case-api.yaml

```diff
@@ -22,6 +22,8 @@ spec:
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/ia/case-api:prod-dbff1f2f
```

Comment:
Don't we need version bump here?

Reply:
nope, version is bumped only for documents api, case-api needs only restart

existing version 0.0.5 matches https://github.com/hmcts/ia-case-api/blob/master/charts/ia-case-api/Chart.yaml#L4
version in application

Reply:
Thanks Marcin..